### ISSUE TITLE
add first tests

### ISSUE THREAD

**Issue by ghost** [PmegaOcalc/primestore#4]
- [x] test for the module to exists

**Comment by PmegaOcalc** [comment]
what module?

**Comment by ghost** [comment]
> what module?

Added this issue just to present the approach:
- Adding the first test to check if our code is there and since it's not there the test would fail. Then to have a passing PR woulda add the following to the requirements of the task.
- [ ] add module

**Comment by PmegaOcalc** [comment]
yes, to write a test to check for a specific module you will have to expect a certain module to exist, what module will you be expecting?

**Comment by ghost** [comment]
The module we are going to implement... Reckon should've used app instead of module!
